docs/architecture/event-flows-proposal.md

```diff
@@ -0,0 +1,85 @@
+# Eventing flow proposal
+
+## Overview
+
+This document shows a proposal of two ways of eventing flow we want to support with use of the [Management Plane](./terminology.md#management-plane)
```

Comment:
This document is a proposal for eventing flows we want to sup..

Reply:
done